app/controllers/api/v1/orders_controller.rb

```diff
@@ -119,14 +120,33 @@ def order_record
           @order.created_by = current_user
           @order.detail_type = "GoodCity"
         end
+
+        if order_params['beneficiary_attributes'] and @order.beneficiary and @order.beneficiary.created_by.nil?
+          # New nested beneficiary
+          @order.beneficiary.created_by = current_user
+        end
+
         @order
       end
 
       def order_params
-        params.require(:order).permit(:stockit_id, :code, :status, :created_at,
-          :organisation_id, :stockit_contact_id, :detail_id, :detail_type, :description,
-          :state, :state_event, :stockit_organisation_id, :stockit_activity_id, :people_helped,
-          :purpose_description, purpose_ids: [], cart_package_ids: [])
+        params.require(:order).permit(
```

Comment:
Much neater, thanks!

Reply:
Yeah, Its clear and readable. Thanks @patrixr :)